nexus/src/app/background/tasks/sync_switch_configuration.rs

```diff
@@ -1338,6 +1338,97 @@ where
     left == right
 }
 
+/// Ensure that a loopback address is created.
+///
+/// loopback_ipv\[46\]_create are not idempotent (see
+/// oxidecomputer/dendrite#343), but this wrapper function is. Call this
```

Comment:
It looks like caller of this function, `add_loopback_addresses_to_switch` is called from a background task, which seems correct. What does the `Call this from sagas instead` indicate? Can you flesh that out a little more here please?

Reply:
Sorry - I meant to, and really should have, called out the code that was moved wholesale from `clients/dpd-client/src/lib.rs` when I deleted it.  I don't actually know off the top of my head what that comment means, since I didn't write the code, and didn't dig into it while moving it.

@jmpesp might be able to explain it faster than I could figure it out?

Reply:
Aha thanks for pointing that out. In that case, I think this PR is good to merge. @jmpesp, can you update the comment at some point?

Reply:
Actually the underlying issue was addressed, this comment and function is outdated :) the dpd endpoints referenced in the issue are now all idempotent.